src/__tests__/a11yAPI.test.js

```diff
@@ -62,74 +64,127 @@ function Section() {
   );
 }
 
-test('getByA11yLabel, queryByA11yLabel', () => {
-  const { getByA11yLabel, queryByA11yLabel } = render(<Section />);
+test('getByA11yLabel, queryByA11yLabel, findByA11yLabel', async () => {
+  const { getByA11yLabel, queryByA11yLabel, findByA11yLabel } = render(
+    <Section />
+  );
 
   expect(getByA11yLabel(BUTTON_LABEL).props.accessibilityLabel).toEqual(
     BUTTON_LABEL
   );
   const button = queryByA11yLabel(/button/g);
   expect(button && button.props.accessibilityLabel).toEqual(BUTTON_LABEL);
+
+  const asyncButton = await findByA11yLabel(BUTTON_LABEL);
+  expect(asyncButton.props.accessibilityLabel).toEqual(BUTTON_LABEL);
+
   expect(() => getByA11yLabel(NO_MATCHES_TEXT)).toThrow(NO_INSTANCES_FOUND);
   expect(queryByA11yLabel(NO_MATCHES_TEXT)).toBeNull();
+  await expect(
```

Comment:
Can we move all async queries to the bottom of these tests? There's a possibility that a sync query would regress and doing async stuff in-between could hide that.

Reply:
Done